docs/result.md

```diff
@@ -1,14 +1,195 @@
-#Return_result
+# Return_result
 
 a POST request is invoked by the client to the server. It returns a JSON file with information like client_id, job, data, and jobs.
 
 ## JSON definition
 
 A job is communicated as a simple JSON object where key names and values are lower case:
 
-* `client_id`: a unique identifier of the client returning the result.
-* `job`: A job object, as defined in job.md. This job corresponds to the result. 
-* `data`: data collected from regulations.gov.
-	* how data is transferred has not been defined. 
-* `jobs`: list of job objects, as defined in job.md.
-	* list of new jobs identified with data. 
+* `client_id`: A unique identifier of the client returning the result.
+* `job_id`: ID of the current job object.
+* `data`: A list of JSON objects. This object consists of a folder name, file name, and data
+  * `folder_name`: The path to the location of where the data should be placed. Check data storage section to see the hierarchy of the path.
+  * `file_name`: Name of the file to which the data will be stored. Named after the type of data it is storing.
+  * `data`: The raw JSON data recieved from querying regulations.gov.
+* `jobs`: A list of new job objects. These job objects omit the job_id field. Jobs are retrieved through the JSON data collected from the regulations server.
+
+## Data Storage
+
+```
+Data (folder)
+  Agency (folder)
+    DocketID (folder)
+      docket.json
+      DocumentID (folder)
+        basic_document.json
+        document.json
+```     
+* Data can be considered the root
+
+```
+EPA/EPA-HQ-OAR-2011-0028/EPA-HQ-OAR-2011-0028-0108/document.json
+
+Agency: EPA
+DocketID: EPA-HQ-OAR-2011-0028
+DocumentID: EPA-HQ-OAR-2011-0028-0108
+File: document.json
+```
+
+
+## Examples
+
+* Docket
+  * Returns data concerning a docket
+  * Placed as a json file in the DocketID folder
+  * Does not contain a `jobs` field
+```
+{
+  'client_id': 'client14',
+  'job_id': 'job33',
+  'data': [
+    {
+      'folder_name': 'EPA/EPA-HQ-OAR-2011-0028/'
+      'file_name': 'docket.json'
+      'data': {
+        'agency': 'EPA',
+        'docket_id': 'EPA-HQ-OAR-2011-0028'
+        'file_contents' = {
+          "agency": "Environmental Protection Agency",
+          "agencyAcronym": "EPA",
+          "cfrCitation": "40 CFR 98",
+        
+          ...
+
+          "internationalImpacts": {
+            "label": "International Impacts",
+            "value": "No"
+          }
+        }
+      }
+    }
+  ]
+}
+```
+
+* Document
+  * Returns data concerning a document
+  * Placed as a json file in the DocumentID folder
+  * `jobs` field can contain a list of download jobs
```

Comment:
Describe where these jobs would be found in the document JSON (in general, not specifically). 

I *believe* they are always in an attachment or file_format section, but verify.  I believe @20ivese knows this information